app/data/0_star_wars.json

```diff
@@ -20,6 +20,530 @@
+                            "id": "ed3e200a-0735-4e8d-9eea-627c1d908697",
+                            "questions": [
+                                {
+                                    "answers": [
+                                        {
+                                            "display": {
+                                                "properties": {
+                                                    "columns": true
+                                                }
+                                            },
+                                            "guidance": "",
+                                            "id": "ca3ce3a3-ae44-4e30-8f85-5b6a7a2fb23c",
+                                            "label": "Choose a side",
+                                            "mandatory": true,
+                                            "options": [
+                                                {
+                                                    "label": "Light Side",
+                                                    "value": "Light Side"
+                                                },
+                                                {
+                                                    "label": "Dark Side",
+                                                    "value": "Dark Side"
+                                                },
+                                                {
+                                                    "label": "I prefer Star Trek",
+                                                    "value": "I prefer Star Trek"
+                                                }
+                                            ],
+                                            "q_code": "20",
+                                            "type": "Radio",
+                                            "validation": {
+                                                "messages": {}
+                                            }
+                                        }
+                                    ],
+                                    "description": "",
+                                    "display": {
+                                        "properties": {}
+                                    },
+                                    "id": "4bfa0229-0ce8-4190-817b-535f55ad2817",
+                                    "title": "",
+                                    "type": "General",
+                                    "validation": []
+                                }
+                            ],
+                            "title": "Choose your side",
+                            "validation": []
+                        }
+                    ],
+                    "routing_rules":[
+                        {
+                            "goto": {
+                                "id" : "96682325-47ab-41e4-a56e-8315a19ffe2a",
+                                "when": {
```

Comment:
Can do, I used 'when' because it made the code look a bit nicer
fugly alert:
 if 'if' not in rule['goto'].keys():